vuepress/src/Documentation/Pre-reading-material/000700_communities_of_practise.md

```diff
@@ -0,0 +1,119 @@
+# Communities of practice
+ ## Value of Communities
+* A strong community will
+    * Provide an anser before the platform team can
+        * Operational cost savings
+        * Faster Service turnarounds
+    * Self-organize to co-create reusable artifacts
+        * Reduce duplication and one-offs
+        * Lighten application maintenance lifecycle requirements
+    * Create Stewards
+        * Early adopters become Influencers
+        * Influencers become Stewards
+
+## Customer Focus – 3 CoPs
+---
+::: tip Acronyms
+
+- **OCIO** = Office of the Associate Deputy Minister and Government Chief Information Officer
+
+- **CoP** = Community of Practise
+
+- **SecOps** = OCIO Security Operations Team
+
```

Comment:
**MISO** = Ministry Information Security Officer, embedded as part of the developer team